modules/capture_zone/functions/CAPZONE/fn_setCaptureZone.sqf

```diff
@@ -58,14 +56,16 @@ private _oldOwner = ["temp", 0, 9999];
 			}
 		};
 
-		if(((_currentOwner select 0) != (_oldOwner select 0)) || (_currentOwner select 0 == "CONTESTED")) then {
+		if(((_currentOwner select 0) != ((GVAR(OLD_OWNERS_MAP) get _marker) select 0)) || (_currentOwner select 0 == "CONTESTED")) then {
```

Comment:
needs a default value for the return since you are using select on the return value

Reply:
[getOrDefault](https://community.bistudio.com/wiki/getOrDefault)

Reply:
> needs a default value for the return since you are using select on the return value

Default value declared in postInitServer.sqf